server/id/id.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+package id
+
+import "context"
+
+// Allocator defines the id allocator on the ceresdb cluster meta info.
+type Allocator interface {
+	// Alloc allocs a unique id.
+	Alloc(ctx context.Context) (uint64, error)
+	// Rebase resets the base for the allocator from the persistent window boundary.
+	Rebase(ctx context.Context) error
```

Comment:
Shall we expose the `Rebase` method to callers? Any cases where callers have to use this method?